replica/backup.go

```diff
@@ -189,6 +189,17 @@ type Hole struct {
 
 var HoleCreatorChan = make(chan Hole, 1000000)
 
+func holeDrainer() {
```

Comment:
duplicate code.. this can be called from Close() of replica/server.go

Reply:
ok